modules/utxo-lib/test/bitgo/signature.ts

```diff
@@ -32,6 +32,8 @@ import { getTransactionWithHighS } from './signatureModify';
 import { normDefault } from '../testutil/normalize';
 import { getKeyName } from '../testutil';
 
+const supportedScriptTypes2Of3 = scriptTypes2Of3.filter((scriptType) => scriptType !== 'p2trMusig2');
+
```

Comment:
```suggestion
function getScriptTypes(): ScriptType2of3[] {
  // FIXME(BG-XXXXX): p2trMusig2 signing does not work in this test suite yet because the test suite is 
  //           written with TransactionBuilder
  return scriptTypes2Of3.filter((scriptType) => scriptType !== 'p2trMusig2');
}
```

Reply:
Done